src/raster/r.random.walk/README.md

```diff
@@ -0,0 +1,11 @@
+# r.random.walk
```

Comment:
No need to duplicate the information in HTML and everything really should be there (is there the reference?). However, I like the inclusion of a Markdown readme file. Let's keep it.

Reply:
That sounds good, but I'm not sure what do you want to me to include in the README?

Reply:
If we want a README.md here, we should recommend that in the `CONTRIBUTING.md` file, too. I think it is a nice idea as it is rendered then well in GitHub. Content could be a one-line and author(s) and the citation, generated with `g.citation`.

Reply:
I don't have any plan for a readme file either, @cwhite911. I guess I was thinking if you have a good use for it without duplication... For r.pops.spread, we copy a README.md file from a separate repo where it has its own purpose which is not applicable here.

@neteler No idea, but we should discuss that more also in context of Markdown docs! ;-)

Reply:
@wenzeslaus I just removed it for now, but I do think it is nice having the readme available for developers that is independent from the html documentation. It could list things like dependencies, information about testing...